README.md

```diff
@@ -1,75 +1,71 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-
-> One paragraph statement about the project.
+To Do list
+> This project for manage To do list .
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+-  HTML,CSS,JavaScript
+-  Visual Studio Code: A source code editor
+-  git: A tool for managing source code
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://mustabbas.github.io/TodoList/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
 
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/mustabbas/AwesomeBooks
+Once the files are on your machine, open the course folder in Visual Studio Code.
 
-### Prerequisites
+With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with Live Server.**
```

Comment:
- [ ] Getting started needs improvement, this is not how we get started with a webpack project.

This is the procedure
1. Clone the repo(**tell us how**)
2. Navigate to project's directory(**tell us how**)
3. Install dependencies(**tell us how**)
4. Lastly start the project(**tell us how**)
    Two options here
     - `npm start` on the terminal OR
     - `npm run build` then navigate to build directory and open the index.html file in the browser